src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -466,3 +494,43 @@ def visit_key(self, node, children):
     def visit_quoted_key(self, node, children):
         # leading ! are used to indicate negation. make sure they don't appear.
         return node.match.groups()[0].lstrip("!")
+
+
+def _load_configs():
+    if not CONFIGS_DIR.exists():
+        logger.warning(
+            "Failed to load Fingerprinting Configs, invalid _config_dir: %s",
+            CONFIGS_DIR,
+        )
+
+    configs = {}
+
+    for config_file_path in sorted(CONFIGS_DIR.glob("**/*.txt")):
+        config_name = config_file_path.parent.name
+        configs.setdefault(config_name, [])
+
+        try:
+            with open(config_file_path) as config_file:
+                str_conf = config_file.read().rstrip()
+                configs[config_name].extend(FingerprintingRules.from_config_string(str_conf).rules)
+        except InvalidFingerprintingConfig as ex:
```

Comment:
Would you want to report this to Sentry? or track it with a metric?

Reply:
This should _not_ be happening in Prod. I've changed it from `warning` to `error` to reflect this